ddl/column_test.go

```diff
@@ -271,9 +271,17 @@ func (s *testColumnSuite) checkColumnKVExist(ctx sessionctx.Context, t table.Tab
 	if err != nil {
 		return errors.Trace(err)
 	}
-	defer ctx.Txn(true).Commit(context.Background())
+	defer func() {
+		if txn, err1 := ctx.Txn(true); err1 == nil {
```

Comment:
```
c.Assert(err1, IsNil)
```
is better?

Reply:
There is no `c.Assert` here, otherwise I've already add it.
I think it doesn't matter since this is in the test code